### PR TITLE
WEBUI-740: limit order input for new vocabulary entries

### PR DESCRIPTION
**Pull request by rakeshkumar1019**
https://jira.nuxeo.com/browse/WEBUI-740

### CODE REVIEW

**Comment by rakeshkumar1019** [comment]
Below are the findings,that supports  for choosing directly number for `max =2147483646` instead of using any language inbuilt property or function.

IN JAVA
 `Integer.MAX_VALUE` represents the maximum positive integer value that can be represented in 32 bits (i.e., 2147483647).
2^31-1 = 2147483647
https://www.educative.io/answers/what-is-integermaxvalue
https://www.geeksforgeeks.org/integer-max_value-and-integer-min_value-in-java-with-examples/

IN JAVASCRIPT

`Number.MAX_VALUE` has the value of 1.7976931348623157e+308.
https://www.geeksforgeeks.org/javascript-number-max_value-property/
* 253-1, or +/- 9,007,199,254,740,991

https://stackoverflow.com/questions/307179/what-is-javascripts-highest-integer-value-that-a-number-can-go-to-without-losin
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER

**Review by GuruprasadAnk1996 — DISMISSED** [review]
looks Good...!!!

**Review by Diksha-Verma-Dev — DISMISSED** [review]

**Review by swarnadipa-dev — DISMISSED** [review]

**Review by richardsd — COMMENTED** [review]
Well, since it seems that there is a mismatch between the Integer max value supported in Java and Javascript, this might be the best option.

File: elements/directory/vocabulary/nuxeo-vocabulary-edit-layout.html [hunk 1]
```diff
@@ -54,13 +54,13 @@
       >
       </paper-toggle-button>
     </div>
-
```
Comment (semisse) [inline elements/directory/vocabulary/nuxeo-vocabulary-edit-layout.html]: Please, do not remove this line

Comment (rakeshkumar1019) [reply elements/directory/vocabulary/nuxeo-vocabulary-edit-layout.html]: sure

**Review by GuruprasadAnk1996 — APPROVED** [review]

**Comment by semisse** [comment]
Please, squash the two commits into one, no need for a separate commit

**Review by Nisha-QA — APPROVED** [review]
Looks good

**Review by SnehalNaik-DEV — APPROVED** [review]
Looks good!

**Review by semisse — APPROVED** [review]

**Pull request merged**